### PR TITLE
Metavisitor2 kickstart.18.05

### PR DESCRIPTION
**Pull request by ccedmendoza**
This changes the version of GalaxyKickstart used to the 18.05 release

### CODE REVIEW

**Comment by drosofff** [comment]
@ccedmendoza I think the error in https://travis-ci.org/ARTbio/metavisitor/jobs/440584514#L932 comes from the fact that you tagged the 18.05 release of GKS on the commit https://github.com/ARTbio/GalaxyKickStart/commit/ae46e888b6dd6ba0953a13fdd579e6e4932c6bc6, which is not really appropriate. Actually there is no reason not to tag on the head of master, which is galaxy_18.05 AND incorporate last improvements. Among which are the pinning of the external roles. I suspected that the requirements.yml here is not compatible anymore with the new mapping of external roles by GalaxyKickStart.

I would recommend to change the current 18.05 GalaxyKickStart release. I don't know if it is possible, but it should.

File: travis_before_install.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ if [ "$TRACK" = "ansible" ]; then
     sudo apt-get -y --purge remove postgresql libpq-dev libpq5 postgresql-client-common postgresql-common
     sudo rm -rf /var/lib/postgresql
     sudo apt-get update -qq
+    ansible-galaxy install -r requirements_roles.yml -p roles
```
Comment (drosofff) [inline travis_before_install.sh:32]: Ah ! I see. No, the idea with metavisitor repo is to provide all roles included, avoiding to fetch independent roles at later stages.
Thus, I start to understand the issue with this PR: The Tag galaxykickstart should include the fetched roles. I don't remember how I did that the first time, but I will probably find it in GKS repo (so do you).

**Comment by drosofff** [comment]
Good news, I see what is wrong. We can probably save the presence of the GKS tar gz archive in this repo, and rely on a distant tar.gz archive in a GalaxyKickStart tagged release. (see also my comment in the code of the PR)

File: build_docker_image.sh [hunk 2]
```diff
@@ -7,13 +7,13 @@ pip --version
...
-tar -xvzf GalaxyKickStart-galaxy_17.09.tar.gz
+tar -xvzf GalaxyKickStart.tar.gz && mv GalaxyKickStart-galaxy_[0-9][0-9].[0-9][0-9] GalaxyKickStart
 rm -rf GalaxyKickStart/Dockerfile GalaxyKickStart/Dockerfile.test
 cp Dockerfile Dockerfile.test GalaxyKickStart/
-rm -rf GalaxyKickStart/group_vars/metavisitor GalaxyKickStart/group_vars/test 
+rm -rf GalaxyKickStart/group_vars/metavisitor GalaxyKickStart/group_vars/test
 cp group_vars/metavisitor group_vars/test GalaxyKickStart/group_vars/
 rm -rf GalaxyKickStart/extra-files/metavisitor GalaxyKickStart/extra-files/test
-cp extra-files/metavisitor extra-files/test GalaxyKickStart/extra-files/
+cp -r extra-files/metavisitor extra-files/test GalaxyKickStart/extra-files/
```
Comment (drosofff) [inline build_docker_image.sh:16]: it is only files (not directories). The -r option is useless

**Comment by drosofff** [comment]
I suggest closing this pr #24 is accepted

**Comment by ccedmendoza** [comment]
Accepting #24 instead

**Pull request closed without merging**